src/auth/user-record.ts

```diff
@@ -240,14 +251,68 @@ export class PhoneMultiFactorInfo extends MultiFactorInfo {
   }
 }
 
+/**
+ * TotpInfo struct associated with a second factor
+ */
+export class TotpInfo {
```

Comment:
Is this neccessary?

Reply:
The backend proto is an empty struct so in case we need to add more information about this later, it might be easier to have a strong type struct than `any` defined in the `MultiFactorInfo` type in case something changes in the future. Happy to reconsider in case there's a suggestion.